win32/PartySample/PartySampleNetworkCommon/lib/NetworkManager.cpp

```diff
@@ -169,10 +169,10 @@ NetworkManager::CreateAndConnectToNetwork(
     PartyNetworkConfiguration cfg = {};
 
     // Setup the network to allow the maximum number of single-device players of any device type
-    cfg.maxDeviceCount = c_maxNetworkConfigurationMaxDeviceCount;
+    cfg.maxDeviceCount = 30;
```

Comment:
nit: your description mentions using 29 but all of your edits use 30 or 32

Reply:
Forget to push last change. Updated.